cz.zeropage.Formiko.json

```diff
@@ -1,19 +1,23 @@
 {
     "app-id": "cz.zeropage.Formiko",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "40",
+    "runtime-version": "42",
     "sdk": "org.gnome.Sdk",
     "command": "formiko",
     "rename-icon": "formiko",
     "rename-desktop-file": "formiko.desktop",
     "rename-appdata-file": "formiko.metainfo.xml",
     "finish-args" : [
         "--share=ipc",
-        "--socket=x11",
+        "--device=dri",
+        "--socket=fallback-x11",
         "--socket=wayland",
         "--share=network",
         "--filesystem=home",
```

Comment:
Is this really needed or can this be reduced to a subset of `$HOME` like XDG directories? I have opened an issue about using portals https://github.com/ondratu/formiko/issues/39

Reply:
I replied to your issue. At this moment it is necessary for preview widget.